<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Pere Constans</author>
 <class>c2bSearchInFilesPattern</class>
 <widget class="QWidget" name="c2bSearchInFilesPattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pattern</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="8">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="c2bLineEdit" name="inputPattern"/>
     </item>
     <item>
      <widget class="QToolButton" name="clearB">
       <property name="icon">
        <iconset resource="cb2bib.qrc">
         <normaloff>:/icons/icons/clear_left.png</normaloff>:/icons/icons/clear_left.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="7">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="NOT">
     <property name="text">
      <string>&amp;NOT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Scope:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="patternType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="caseSensitive">
     <property name="text">
      <string>&amp;Case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QComboBox" name="yearScope">
     <item>
      <property name="text">
       <string>Exact</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Newer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Older</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QComboBox" name="patternScope">
     <item>
      <property name="text">
       <string>all</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>c2bLineEdit</class>
   <extends>QLineEdit</extends>
   <header>c2bLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cb2bib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearB</sender>
   <signal>clicked()</signal>
   <receiver>inputPattern</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearB</sender>
   <signal>clicked()</signal>
   <receiver>inputPattern</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
